packages/route-viewer-overlay/src/index.js

```diff
@@ -16,6 +17,28 @@ const isGeomComplete = routeData => {
   );
 };
 
+// helper function that removes all points from array of points that are
+// within flex zones defined in an array of stops
```

Comment:
Use JSDoc style `/**` comment.

Reply:
Done in 6945aafe0033c070e7556d47894b2c7de63b4441